src/foam/u2/layout/Card.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.layout',
+  name: 'Card',
+  extends: 'foam.u2.borders.CardBorder',
+
+  properties: [
+    {
+      class: 'Int',
+      name: 'def',
+      value: 12
+    },
+    {
+      class: 'Int',
+      name: 'md',
+      expression: function(def) {
+        return def;
+      }
+    },
+    {
+      class: 'Int',
+      name: 'lg',
+      expression: function(def) {
+        return def;
+      }
+    },
+    {
+      class: 'Int',
+      name: 'sm',
+      expression: function(def) {
+        return def;
+      }
+    }
+  ],
+
+  methods: [
+    function initE() {
+      this.SUPER();
+      this.addClass(this.myClass());
+
+      // TODO: find the display mode and adjust accordingly (e.g. for sm divide by 8), for now just use md
```

Comment:
Let's put some of this stuff in place now. Import 'displayWidth?' and make decisions based on that. Create an enum that contains the widths we'll be caring about and assume displayWidth is one of those, or give it a default if it's undefined. Make sure this initE is bound to the displayWidth because it can/will change.

Reply:
Done.